src/type/preload.ts

```diff
@@ -494,6 +496,7 @@ export type SplitterPosition = z.infer<typeof splitterPositionSchema>;
 
 export type ConfirmedTips = {
   tweakableSliderByScroll: boolean;
+  noticeAltPortInfo: boolean;
```

Comment:
これ増えてくるとどれがどれかわからなくなりそうですね！！
メモ追加お願いできると･･･！
```suggestion
  noticeAltPortInfo: boolean; // エンジンのポート変更の通知
```